jobs/rocoto/wavepostbndpnt.sh

```diff
@@ -19,6 +21,8 @@ echo "=============== START TO RUN WAVE_POST_BNDPNT ==============="
 # Execute the JJOB
 ${HOMEgfs}/jobs/JGLOBAL_WAVE_POST_BNDPNT
 status=$?
-[[ ${status} -ne 0 ]] && exit ${status}
+if [[ ${status} -ne 0 ]]; then
+    exit "${status}"
+fi
 
 exit 0
```

Comment:
Similar question about when to use `exit 0` versus `exit "${status}"` at the end of the rocoto job scripts.

Reply:
Same and done.